packages/server/lib/browsers/firefox.ts

```diff
@@ -412,5 +414,38 @@ export async function open (browser: Browser, url, options: any = {}) {
 
   await firefoxUtil.setup({ extensions: launchOptions.extensions, url, foxdriverPort, marionettePort })
 
+  // Override the .kill method for Windows so that the Browsers closes b/w specs
+  if (os.platform() === 'win32') {
+    browserInstance.kill = async (...args) => {
```

Comment:
```suggestion
    browserInstance.kill = (...args) => {
```

child_process.kill isn't async is it?